src/ai/backend/client/request.py

```diff
@@ -573,6 +573,7 @@ def __init__(
     async def __aenter__(self) -> Response:
         max_retries = len(self.session.config.endpoints)
         retry_count = 0
+        raw_resp = None
```

Comment:
How about add a type hint here?

Reply:
i added type hint like other part in that file.